backend/main_fixed.py

```
@@ -621,11 +700,16 @@ async def detect_garbage_endpoint(image: UploadFile = File(...)):
         logger.error(f"Garbage detection error: {e}", exc_info=True)
-        raise HTTPException(status_code=500, detail="Detection service temporarily unavailable")
+        raise HTTPException(
+            status_code=500, detail="Detection service temporarily unavailable"
+        )
+
 
 # External API Detection Endpoints (HuggingFace CLIP-based)
 @app.post("/api/detect-illegal-parking")
-async def detect_illegal_parking_endpoint(request: Request, image: UploadFile = File(...)):
+async def detect_illegal_parking_endpoint(
+    request: Request, image: UploadFile = File(...)
```

Ruff (0.15.12)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)